includes/class-wc-payments-onboarding-service.php

```diff
@@ -72,7 +72,8 @@ class WC_Payments_Onboarding_Service {
 	const FROM_STRIPE_EMBEDDED  = 'STRIPE_EMBEDDED';
 	const FROM_REFERRAL         = 'REFERRAL';
 
-	const TRACKS_EVENT_ONBOARDING_RESET = 'wcpay_onboarding_flow_reset';
+	const TRACKS_EVENT_ONBOARDING_RESET   = 'wcpay_onboarding_flow_reset';
+	const TRACKS_EVENT_DISABLE_TEST_DRIVE = 'wcpay_test_drive_account_disabled';
```

Comment:
I would rename this to `TRACKS_EVENT_TEST_DRIVE_ACCOUNT_DISABLE` and the event name to `wcpay_onboarding_test_account_disable`

Reply:
Makes sense, updated!